### PR TITLE
Update Example code: resolved error of initialized var of value to isInitialized

### PR DESCRIPTION
**Pull request by krunal704**

### CODE REVIEW

File: just_audio/example/lib/main.dart [hunk 1]
```diff
@@ -71,7 +71,7 @@ class _MyAppState extends State<MyApp> {
   }
 
   Future<void> _init() async {
-    final session = await AudioSession.instance;
+    final session = await AudioSession.isInitialized;
```
Comment (ryanheise) [inline just_audio/example/lib/main.dart:74]: I do not understand this pull request as I haven't provided any getter in audio_session called `isInitialized`.

Comment (krunal704) [reply just_audio/example/lib/main.dart:74]: Ohh, Yes you are correct. It was supposed to be on different repository, but by mistake I did on this repo.

Sorry

Comment (ryanheise) [reply just_audio/example/lib/main.dart:74]: Ah, OK :-) Mystery solved.

Comment (krunal704) [reply just_audio/example/lib/main.dart:74]: :smiley:

**Pull request closed without merging**